### PR TITLE
adding Artwork metadata component

### PR DESCRIPTION
**Pull request by oxaudo**
Another baby step in the sidebar.

Produced this:
screen shot 2018-06-12 at 2 15 56 pm

Thanks @zephraph for walking me through `styled-system` conventions.

Also not exactly sure level of stories is good there. Quite a lot of data setup but I think it's good to see all the variations. And also I think once we start testing with real data - would be very helpful to produce the extreme case data scenarios and have those displayed in stories

### CODE REVIEW

File: src/Styleguide/Pages/Artwork/Sidebar/Artists.tsx [hunk 1]
```diff
@@ -42,16 +53,12 @@ export class Artists extends React.Component<ArtistsProps> {
...
-        <Artist>
-          {this.renderArtistName(artist)}
-          <FollowIcon is_followed={artist.is_followed} />
-        </Artist>
-      )
-    } else {
-      return <ArtistsContainer>{this.renderArtists(artists)}</ArtistsContainer>
-    }
+    return (
+      <ArtistsContainer pb={4}>
```
Comment (damassi) [inline src/Styleguide/Pages/Artwork/Sidebar/Artists.tsx]: `pb` 😄 its nice

File: src/Styleguide/Pages/Artwork/Sidebar/ArtworkMetadata.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  ${space};
+`
+
+export class ArtworkMetadata extends React.Component<ArtworkMetadataProps> {
+  render() {
+    const { artwork } = this.props
+    return (
+      <ArtworkMetadataContainer pb={4}>
+        <TitleInfo artwork={artwork} />
+        {artwork.edition_sets.length > 1 ? "" : <SizeInfo artwork={artwork} />}
```
Comment (damassi) [inline src/Styleguide/Pages/Artwork/Sidebar/ArtworkMetadata.tsx]: Can do 
```jsx
{artwork.edition_sets.length > 1 &&
  <SizeInfo artwork={artwork} />
}
```
As falsy items won't render

File: src/Styleguide/Pages/Artwork/Sidebar/SizeInfo.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  render() {
+    const { artwork } = this.props
+    return (
+      <SizeInfoContainer>
+        <Serif size="2">
+          {artwork.dimensions
+            ? [artwork.dimensions.in, artwork.dimensions.cm].join("; ")
+            : ""}
+        </Serif>
+        {artwork.edition_of ? <Serif size="2">{artwork.edition_of}</Serif> : ""}
```
Comment (damassi) [inline src/Styleguide/Pages/Artwork/Sidebar/SizeInfo.tsx]: These two spots can also take advantage of `&&` as opposed to a ternary

**Comment by oxaudo** [comment]
Merging this.

**Pull request merged**